source/administration/hipchat-migration-guidelines.rst

```diff
@@ -43,5 +43,25 @@ Step 3: Import your data into Mattermost
 
 2. Alternatively, `contact Mattermost <https://mattermost.com/contact-us/>`_ for partner recommendations for your region to assist in your import. 
   
+Step 4: Onboard your users into Mattermost
+---------------------------------------------
+After importing users, you can send out an announcement via email or via your old system (or both) to let users know how to log into Mattermost with their old accounts or how to create new accounts.
+ 
+**Announcing Mattermost onboarding in your previous messaging system:**
+ 
+Use the following message template to alert users of the migration:
 
+  @all, we’re moving communications to a new Mattermost server at [your new location, e.g. ``https://yourcompany.com/mattermost``]. You can start your new account by going to the page, clicking on **I forgot my password** and entering the email you used on this system in the Reset Password page to set up new credentials. Your message history and channels should carry over from this system into Mattermost. Any questions? Please let us know.
 
```

Comment:
"You can start your new account by going to the page" -> can we specify what page?